pom.xml

```diff
@@ -2863,6 +2894,7 @@
         <artifactId>maven-shade-plugin</artifactId>
         <configuration>
           <shadedArtifactAttached>false</shadedArtifactAttached>
+          <createDependencyReducedPom>false</createDependencyReducedPom>
```

Comment:
This is to avoid a [potential bug](https://issues.apache.org/jira/browse/MSHADE-148) in Maven where it could stuck in an infinite loop when generating reduced pom.